source/manuals/monitor-paas-app-with-prometheus.html.md.erb

```diff
@@ -39,21 +39,98 @@ You can create a Prometheus service within your PaaS space and allow it to bind
 
 Within 5 minutes Prometheus will start scraping your application for metrics, you can validate this by checking [Grafana][].
 
-### App route
+### App route configuration
 
 Whether or not you're using a custom domain the Prometheus service broker will only scrape your app's first route.
 
 For example, some apps may have multiple routes especially if they use custom domains such as `custom.domain.gov.uk, an-interesting-app.cloudapps.digital`. Only the first route will be picked up.
 
-If there are no routes to your app the Prometheus service will default the route to `<app-name>.cloudapps.digital`.
+If there are no routes to your app the Prometheus service will default the route to:
+
+`<app-name>.cloudapps.digital`.
 
 Because changes to the first route will only be picked up within 5 minutes it could create gaps to metrics during route changes if you're using a zero downtime plugin such as [autopilot][].
 
+## Configure container metrics
+
+[Cloud Foundry][] provides time-series data (metrics), for your PaaS apps.
+
+Use the metrics exporter app to send Cloud Foundry metrics data to a monitoring system of your choice to store and view it. You'll need to deploy the metrics exporter as an app on PaaS before you can use it.
+
+Currently suppprted metrics are:
+
+* CPU
+* RAM
+* disk usage data
+* app crashes
+* app requests
+* app response times
+
+### Set up the metrics exporter app
+
+Before you setting up the metrics exporter app, you'll need a live Cloud Foundry account assigned to the spaces you want to receive metrics on.
```

Comment:
Is a link or any guidance we can link to that explains how to get a CF account?

Reply:
I'll add a link